### DISCUSSION TITLE
Offset sheet from top / bottom / etc.

### DISCUSSION THREAD

**Discussion by cem-al** [shadcn-ui/ui#2499] [Q&A]
Is it possible to offset the sheet component so it slides in from say 100px from the top of the page?

 I've looked into `tailwind-animate` and have tried applying a number to the `data-[state=open]:slide-in-from-top-100` but this does not work.

**Comment by edlucas** [comment]
I've tried to solve this problem by adding a container to `<SheetPortal>` directly (in the component `sheet.tsx`), but that does not seem to work. For example:

```jsx
<SheetPortal container={document.getElementById('testRoot')}>
```

I got this from the Radix docs for the Dialog component: https://www.radix-ui.com/primitives/docs/components/dialog#portal
